lib/base-compiler.js

```diff
@@ -140,16 +141,16 @@ export class BaseCompiler {
                 };
             }
         });
-    
+
```

Comment:
You (or maybe your editor) are right that these space should be removed. But maybe we could do that in an another commit ?

Reply:
Oh didn't notice this. It's quite distracting. I'll separate it in different commits.

Reply:
Thanks!